vpi/common_vpi.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#include <inttypes.h>
+#include <vpi_user.h>
+
+/*
+ * register_cb is a wrapper function around vpi_register_cb
+ * to simplify the the callback registering process
+ */
+void register_cb(PLI_INT32(*f)(p_cb_data),
+                 PLI_INT32 reason,
+                 int64_t cycles){
+
+    s_cb_data cbData;
```

Comment:
I think that this deserves a reference/cite, either here or in the docs. Where should users search for the definition of `s_cb_data`? In `vpi_user.h`? The standard? Some book?

Reply:
Now explained in #21.